data-model/src/main/scala/za/co/absa/enceladus/model/properties/propertyType/package.scala

```diff
@@ -37,7 +37,11 @@ package object propertyType {
     override def isValueConforming(value: String): Try[Unit] = Success(Unit)
   }
 
-  case class EnumPropertyType(allowedValues: Set[String], suggestedValue: Option[String]) extends PropertyType {
+  case class EnumPropertyType(allowedValues: Seq[String], suggestedValue: Option[String]) extends PropertyType {
```

Comment:
Could we use `LinkedHashSet`?

Reply:
Solved by require on `size=toSet.size`, thanks for the hint.